frontend/src/pages/auth/RegisterForm.jsx

```diff
@@ -49,10 +50,21 @@ return (
                 <button className="buttonProfilePic5" value="5" onClick={(e) => setProfileId(e.target.value)}></button>
                 {console.log(profileId)}
             </div>
+            <div className="form-group">
+                <select id = "dropdown" onChange={(e) => setFoodId(e.target.value)}>
+                    <option value="0">Select your food preference:</option>
+                    <option value="1">Meat</option>
+                    <option value="2">Chicken</option>
+                    <option value="3">Fish</option>
+                    <option value="4">Vegetarian</option>
+                    <option value="5">Vegan</option>
```

Comment:
Here as well.

Reply:
this one is on the registration page, so there shouldn't be any issues 👍

Reply:
![Screenshot 2021-05-07 232330](https://user-images.githubusercontent.com/79463176/117538368-23b09700-b006-11eb-8d0a-d8e00be7557c.png)